.github/workflows/build.yml

```
@@ -204,3 +204,9 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Upload docs
+        uses: actions/upload-artifact@v2.2.1
```

```suggestion
        uses: actions/upload-artifact@v2.3.0
```

You're using an old version of the action, you might want to update to the latest version?